src/templates/docs/font-family/index.html

```diff
@@ -47,26 +47,37 @@ <h4 class="f4 mv0 fw6 dib mr4"><%= name %></h4>
       <div class="ph3 ph5-ns pt4 pb5">
         <h2 class="f3 bold">Examples</h2>
         <h3 class="f5 book sans-serif pt4 caps">Default System Sans Serif</h3>
+        <code class="db f6">.system-sans-serif</code>
         <p class="system-sans-serif lh-copy measure">
           Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
           tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
           vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren,
           no sea takimata sanctus est Lorem ipsum dolor sit amet.
         </p>
         <h3 class="f5 book sans-serif pt4 caps">Default System Serif</h3>
+        <code class="db f6">.system-serif</code>
         <p class="system-serif lh-copy measure">
           Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
           tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
           vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren,
           no sea takimata sanctus est Lorem ipsum dolor sit amet.
         </p>
         <h3 class="f5 book sans-serif pt4 caps">System Sans Serif with Fallbacks</h3>
+        <code class="db f6">.sans-serif</code>
         <p class="sans-serif lh-copy measure">
           Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
           tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
           vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren,
           no sea takimata sanctus est Lorem ipsum dolor sit amet.
         </p>
+        <h3 class="f5 book sans-serif pt4 caps">Serif with Fallbacks</h3>
```

Comment:
Added the `serif` example, users may miss that its a possibility if they don't check out the source code snippet at the bottom of the page